### ISSUE TITLE
ClemCoin

### ISSUE THREAD

**Issue by zepthro** [ClemsonCPSC-Discord/ClemBot#339]
## What is ClemCoin?
ClemCoin is a currency. The currency is based on your local Clembot you are running, or we could have a global currency? No matter. I think local currency is the best option for the first implementation.

## Why ClemCoin?
I want to add ClemCoin for the newest feature I am working on, "gamecenter". The gamecenter is a cog that has various casino style games. For example, texas hold'em, and blackjack. ClemCoin would enhance the gamecenter games by adding a no risk gambling aspect.

## Possible enchancements
ClemCoin has the possibility to remove cooldowns for popular cogs like **slots**, and **ogslots**. With ClemCoin you can add a minimum bet to play the cog. Now instead of a time constraint for Clembot users, they have a "currency constraint".

## Topics of discussion
First idea: **Daily ClemCoin + Nitro Boosters get a multiplier for daily ClemCoins.**
Second idea: certain cogs, or activities give you a base amount of ClemCoins. 

The first idea is the easiest to implement, but I like parts of the second idea. The second idea forces users to interact with parts of Clembot. We can even add daily bonuses to areas we want users to interact with.

local based vs global based vs both?

## tl;dr
ClemCoin is a currency. ClemCoin will add various ways we can enhance Clembot. I need your opinion on wealth creation.

**Comment by Exper1mental** [comment]
You could possibly have a mixture of global and server-based currencies with exchange rates for currencies if you wanted to get fancy.

I like the idea of introducing a currency to eliminate the cooldown. Obviously, the price of slots would need to be set so that in general you lose money-spinning, requiring the acquisition of more currency elsewhere.
